packages/@react-aria/collections/src/ScrollView.tsx

```diff
@@ -165,8 +166,9 @@ function ScrollView(props: ScrollViewProps, ref: RefObject<HTMLDivElement>) {
     style.overflow = 'auto';
   }
 
+  let {focusedKey, isLoading, ...otherotherProps} = otherProps;
```

Comment:
haven't dug too deep here, but could we call `filterDOMProps(otherProps)` on the div below instead of doing the above? If there are some exceptions that we want to let through to the div, I guess we could pass a override object to filterDOMProps.

Reply:
when i used the default filter, it caused more issues, but i can look more into those now that it's mostly settled down and i won't be conflating issues